go/datas/commit_test.go

```diff
@@ -82,3 +84,57 @@ func TestCommitWithoutMetaField(t *testing.T) {
 	})
 	assert.False(IsCommitType(noMetaCommit.Type()))
 }
+
+func TestCommitDescendsFrom(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
PTAL @rafael-atticlabs. I've restructured the tests in the spirit of your comment.